config/settings.yml

```diff
@@ -2,7 +2,8 @@
 ARCGIS_BASE_URL: "https://www.arcgis.com/home/webmap/viewer.html"
 
 # DOWNLOAD_PATH is set in production but defaults to this tmp dir when not set
-# DOWNLOAD_PATH: './tmp/cache/downloads'
+DOWNLOAD_PATH: './tmp/cache/downloads'
+download_cache_expiry_time_days: 14 # the number of days since a download was last accessed before it is removed from the cache
 
```

Comment:
new setting for expiry time and explicitly make it clear what the download path is in development (it is overriden by shared configs for all deployed environments)